Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/nhin/InboundDocRetrieveStrategyImpl.java

```diff
@@ -121,9 +121,9 @@ public RetrieveDocumentSetResponseType sendToAdapter(InboundDocRetrieveOrchestra
     public void auditInboundResponseMessage(InboundDocRetrieveOrchestratable message) {
 
         LOG.debug("Calling audit log for doc retrieve response received from adapter (a0)");
-        docRetrieveLogger.auditResponseMessage(message.getRequest(), message.getResponse(), message.getAssertion(),
-            null, NhincConstants.AUDIT_LOG_INBOUND_DIRECTION, NhincConstants.AUDIT_LOG_ADAPTER_INTERFACE,
-            Boolean.FALSE, message.getWebContextProperties(), NhincConstants.DOC_RETRIEVE_SERVICE_NAME);
+//        docRetrieveLogger.auditResponseMessage(message.getRequest(), message.getResponse(), message.getAssertion(),
```

Comment:
Always follow this rule when submitting a PR with commented-out code:

Make sure to add a comment above the code as to why it wasn't removed.  If you can't come up with a reason, remove the code instead.